src/main/resources/templates/fhirvalidationreport/fhir_validation_report.ftl

```diff
@@ -16,25 +16,12 @@
             <#else>
                 <div class="report-body">
                     <#include "validation_report_header.ftl" />
-                    <#if measureFhirValidationErrors?has_content
-                        || valueSetFhirValidationErrors?has_content
-                        || libraryFhirValidationErrors?has_content>
-                        <div class="custom-card">
-                            <div class="card-header">FHIR Resources to Reconcile</div>
-                            <#if measureFhirValidationErrors?has_content>
-                                <#include "measure_fhir_validation_errors.ftl" />
-                            </#if>
-                            <#if valueSetFhirValidationErrors?has_content>
-                                <#include "valueset_fhir_validation_errors.ftl" />
-                            </#if>
-                            <#if libraryFhirValidationErrors?has_content>
-                                <#include "library_fhir_validation_errors.ftl" />
-                            </#if>
-                        </div>
-                    </#if>
-                    <#if cqlConversionErrors?has_content>
-                        <#include "cql_conversion_errors.ftl" />
-                    </#if>
+<#--                    <#include "fields_to_reconsile.ftl" />-->
```

Comment:
is it temporarily commented out?

Reply:
Yes, Fields to reconcile data is not being received from MicroServices yet.